lgn-messages/src/types/v1/preprocessing/ext_tasks.rs

```diff
@@ -279,7 +279,7 @@ impl FinalExtraction {
                 mapping_table_hash,
                 block_nr,
                 contract,
-                Some(TableDimension::Single),
+                Some(TableDimension::Compound),
```

Comment:
That naming is very confusing lol i understand why this was overlooked:
* `SingleTableExtraction`
* `TableDimension::Compound`
The fact we have the two intertwined like this is weird. I think `TableExtraction` should be sufficient to identify what thsi struct is about and that this table has a dimension. 

https://github.com/Lagrange-Labs/mapreduce-plonky2/blob/feat/merge_table/mp2-v1/tests/common/cases/table_source.rs#L991